treeherder/webapp/api/auth.py

```diff
@@ -16,3 +27,28 @@ def hawk_lookup(id):
         'key': str(credentials.secret),
         'algorithm': 'sha256'
     }
+
+
+class TaskclusterAuthViewSet(viewsets.ViewSet):
+
+    @list_route(methods=['get'])
```

Comment:
`methods=['get']` is redundant, since it's the default (https://github.com/tomchristie/django-rest-framework/blob/3.3.3/rest_framework/decorators.py#L132), however if you'd prefer to include it for clarity, that's fine too.